internal/colorize/cropped.go

```diff
@@ -0,0 +1,58 @@
+package colorize
+
+type Entry struct {
+	Line   string
+	Length int
+}
+
+func GetCroppedText(text string, maxLen int) []Entry {
+	entries := make([]Entry, 0)
+	entryText := ""
+	pos := 0
+	count := 0
```

Comment:
The difference between these is not clear from their name. Please name them a little more explicitly.